package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-slingshot",
-  "version": "5.0.0",
+  "version": "7.0.0",
```

Comment:
Yeah, I forgot the current release was 6.0, so I was mostly trying to figure how one change was 2 version. All good by me.